Source/DafnyDriver/DafnyDriver.csproj

```diff
@@ -1,18 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-      <OutputType>Exe</OutputType>
-      <AssemblyName>Dafny</AssemblyName>
+      <OutputType>Library</OutputType>
```

Comment:
One nice impact of this is that a client can now get _all_ of Dafny as a library (though only conveniently if we publish this library to NuGet).

Reply:
Yes, and I think I'm going to have to define an additional "DafnyWithoutRuntimes" Exe project that depends on this in order to run Dafny on my `DafnyRuntime.dfy` code in the next PR. My intent remains to keep this and that project internal-only for now though.